backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -410,14 +393,22 @@
   ::data)
 
 (defmethod ig/init-key :akvo.lumen.component.keycloak/keycloak  [_ {:keys [credentials data] :as opts}]
+  (log/info "Starting keycloak")
   (let [{:keys [issuer openid-config api-root] :as this} (keycloak (assoc data :credentials credentials))
         http-timeout 10000
-        openid-config (fetch-openid-configuration issuer {:timeout http-timeout})]
+        cm (http.conn-mgr/make-reusable-conn-manager {:timeout 2 :threads 3 :insecure? false :default-per-route 10})
```

Comment:
`threads` is actually the max number of connections for all routes, while `default-per-route` is the maximum connections for a given route. Note that route here means host, which means the keycloak server.

Given that the pool is going to be used for one route (the keycloak host), `thread` and `default-per-route` should be the same. 10 seems like a sensible value.